tools/view_model.cpp

```diff
@@ -465,25 +453,17 @@ int main(int argc, char **argv)
         else if (key == 'p')
         {
             // Snap pitch to 90 degrees
-//            if (cam_pitch < -0.785)
-//                cam_pitch = -1.57;
-//            else if (cam_pitch > 0.785)
-//                cam_pitch = 1.57;
-//            else
-//                cam_pitch = 0;
-
-            if (cam_pitch > 0)
-                cam_pitch = (int)(cam_pitch / 1.57 + 0.5) * 1.57;
+            if (cam_pitch < 1.57)
+                cam_pitch = std::round(cam_pitch / 1.57 + 0.51) * 1.57;
             else
-                cam_pitch = (int)(cam_pitch / 1.57 - 0.5) * 1.57;
+                cam_pitch = std::round(cam_pitch / 1.57 - 0.51) * 1.57;
 
-            // Snap yaw to 90 degrees
-            if (cam_yaw > 0)
-                cam_yaw = (int)(cam_yaw / 1.57 + 0.5) * 1.57;
-            else
-                cam_yaw = (int)(cam_yaw / 1.57 - 0.5) * 1.57;
+//            // Snap yaw to 90 degrees
+//            if (cam_yaw < 0)
+//                cam_yaw = std::round(cam_yaw / 1.57 + 0.51) * 1.57;
+//            else
+//                cam_yaw = std::round(cam_yaw / 1.57 - 0.51) * 1.57;
```

Comment:
Explain why it's disabled?, or remove it?

Reply:
removed